crates/bevy_log/src/lib.rs

```diff
@@ -112,23 +111,24 @@ pub struct LogPlugin {
     /// This can be further filtered using the `filter` setting.
     pub level: Level,
 
-    /// Optionally apply extra transformations to the tracing subscriber,
-    /// such as adding [`Layer`](tracing_subscriber::layer::Layer)s.
+    /// Optionally add an extra [`Layer`] to the tracing subscriber
+    ///
+    /// (`Vec<Layer>` is also accepted, as it implements [`Layer`]).
     ///
     /// Access to [`App`] is also provided to allow for communication between the [`Subscriber`]
     /// and the [`App`].
```

Comment:
```suggestion
    /// Optionally add an extra [`Layer`] to the tracing subscriber
    ///
    /// This function is only called once, when the plugin is built.
    ///
    /// Because [`BoxedLayer`] takes a `dyn Layer`, `Vec<Layer>` is also an acceptable return value.
    ///
    /// Access to [`App`] is also provided to allow for communication between the [`Subscriber`]
    /// and the [`App`].
    ///
    /// Please see the `examples/log_layers.rs` for a complete example.
```

I feel like the docs here can be improved, since this feature is somewhat more complex.